pkg/services/vcs/service/service_mock.go

```diff
@@ -0,0 +1,45 @@
+package Service
```

Comment:
Do we really need to create a new package? The service will be fairly small so I think it can go into the vcs package directly.

Reply:
We usually do in my team, to respect the standard package layout. But I won't have any hard feeling if we don't :wink: